sys/parsoid.js

```diff
@@ -318,7 +318,8 @@ class ParsoidService {
                             // This revision is actually a redirect. Pass redirect target
                             // to caller, and let it rewrite the location header.
                             resp.status = 302;
-                            resp.headers.location = encodeURIComponent(redirectTarget);
+                            // Filter will take care of uri-encoding
+                            resp.headers.location = redirectTarget;
```

Comment:
This strikes me as a bit brittle. Is this mainly meant as a performance optimization, or is there another reason to defer encoding?